Header header               # message header
string sender_st            # name of sender
int8 type_i                 # motion vector type
int8 prio_i                 # message priority
float32 x_f                 # x coordinate
float32 y_f                 # y coordinate